pkg/injector/service/injector.go

```diff
@@ -110,29 +129,28 @@ func getAppIDFromRequest(req *admissionv1.AdmissionRequest) (appID string) {
 }
 
 // NewInjector returns a new Injector instance with the given config.
-func NewInjector(authUIDs []string, config Config, daprClient scheme.Interface, kubeClient kubernetes.Interface) (Injector, error) {
+func NewInjector(opts Options) (Injector, error) {
 	mux := http.NewServeMux()
 
 	i := &injector{
-		config: config,
+		config: opts.Config,
 		deserializer: serializer.NewCodecFactory(
 			runtime.NewScheme(),
 		).UniversalDeserializer(),
 		server: &http.Server{
-			Addr:    fmt.Sprintf(":%d", port),
-			Handler: mux,
-			TLSConfig: &tls.Config{
-				MinVersion: tls.VersionTLS12,
```

Comment:
This needs to be kept for all control plane services

Reply:
Done in the spiffe library.